README.md

```diff
@@ -168,6 +168,30 @@ public class Handler implements RequestHandler<APIGatewayV2ProxyRequestEvent, AP
 }
 ```
 
+### Trace/Log Correlations
+
+In order to correlate your traces with your logs, you must inject the Trace ID (Datadog or XRay)
```

Comment:
Nit: XRay -> X-Ray

Also, I thought you always need to inject the **Datadog** trace ID (even if using X-Ray), hence the logic for conversion from the X-Ray to Datadog trace ID.

Reply:
Other nit: Trace ID -> trace ID throughout

Reply:
Good point. To be more clear, I should say "trace ID (regardless of whether you're using Datadog or X-Ray)

Reply:
@DarcyRaynerDD , to confirm, the span ID is the parent ID, correct?